error.vue

```diff
@@ -12,16 +12,16 @@
               Or as we like to say, <span v-html="randomMessage" />
             </p>
             <UiCta
-              :label="backToHomeCta.label"
-              :segment="backToHomeCta.segment"
-              :url="backToHomeCta.url"
-            />
-            <UiCta
-              class="error-page__ghost-btn"
+              v-if="isDocumentationError"
```

Comment:
Instead of doing an `if-else` here, you can calculate the CTA to show in a new variable. So we will have something like this:

```html
<UiCta
    :label="backToCta.label"
    :segment="backToCta.segment"
    :url="backToCta.url"
  />
```

```js

const backToCta = computed<Link>(() => route.path.startsWith("/documentation/") 
? backToDocsCta 
: backToHomeCta);
```

Please, review types and naming because I did it quickly.

One more thing, I've removed the check for the 404 page, because I supposed we also want to have the `Back to` CTA with no matter which error, right?

Reply:
updated [here](https://github.com/Qiskit/qiskit.org/pull/3369/commits/081fc828d249ea0ef09140593eee261683462a35)